dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/imports/validation/validator/event/DataValuesValidatorTest.java

```diff
@@ -783,9 +781,73 @@ void failValidationDataElementOptionValueIsInValid()
     }
 
     @Test
-    void failValidationWhenOrgUnitValueIsInvalid()
+    void successValidationDataElementMultiTextOptionValueIsValid()
     {
+        DataValue validDataValue = dataValue( "CODE,CODE1" );
+        DataValue nullDataValue = dataValue( null );
+
+        OptionSet optionSet = new OptionSet();
+        Option option = new Option();
+        option.setCode( "CODE" );
+        Option option1 = new Option();
+        option1.setCode( "CODE1" );
+        optionSet.setOptions( Arrays.asList( option, option1 ) );
```

Comment:
nitpick: we should prefer `List.of` when possible see https://github.com/dhis2/wow-backend/blob/master/guides/java11_pitfalls.md#listof-as-a-replacement-for-arraysaslist-and-alike

Reply:
ok, good. replaced.